zulipterminal/ui_tools/boxes.py

```diff
@@ -403,12 +408,51 @@ def stream_box_view(
         )
         self.stream_write_box.set_completer_delims("")
         self._setup_common_stream_compose(stream_id, caption, title)
-
         # Use and set a callback to set the stream marker
         self._set_stream_write_box_style(None, caption)
         urwid.connect_signal(
             self.stream_write_box, "change", self._set_stream_write_box_style
         )
+        self.check_stream_topic_edit_txt(self.check_stream_topic_edit_txt_flag)
+
+    @asynch
+    def check_stream_topic_edit_txt(
+        self, check_stream_topic_edit_txt_flag: bool
+    ) -> None:
+        narrow_key = primary_key_for_command("NARROW_MESSAGE_RECIPIENT")
+        narrow_markup = [
+            ("footer", "Help(?): "),
+            ("footer_contrast", f" {narrow_key} "),
+            ("footer", " Narrow to conversation."),
```

Comment:
This styling (and below) is extracted from the other location. It would be better to use a common functionality.